src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.ts

```diff
@@ -61,7 +61,7 @@ export const useLensSuggestions = ({
   const [allSuggestions, setAllSuggestions] = useState(suggestions.allSuggestions);
   const currentSuggestion = originalSuggestion ?? suggestions.firstSuggestion;
   const suggestionDeps = useRef(getSuggestionDeps({ dataView, query, columns }));
-
+  const histogramQuery = useRef<string>('');
```

Comment:
```suggestion
  const histogramQuery = useRef<AggregateQuery>();
```
Could we store the whole query object in the ref instead to avoid returning a new object reference on each render? Just thinking it could trigger some unwanted effects given the memoization we use elsewhere in Unified Histogram.

Reply:
Thanx my dear Davis. Addressed here https://github.com/elastic/kibana/pull/169911/commits/d94f1a481670ea70f0ea6598bd38631c9481a5af